### PR TITLE
Feat/submit completions

### PR DESCRIPTION
**Pull request by dannil76**
## Explain the changes you’ve made
Added check if current from id is random check or normal completion form type

## Explain why these changes are made
These changes completes the completion flow

## Tested environments
- [x] Your personal AWS environment.

### CODE REVIEW

**Comment by sebastianthulin** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/helsingborg-stad/repositories/602c13c73b93a7537d767e421a43afaa/pull-request/298)

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1pdu5dc Logik för att hantera bilder som ska skickas till Viva med kategori - Komplettering](https://app.clickup.com/t/1pdu5dc)

File: services/viva-ms/lambdas/submitCompletion.js [hunk 1]
```diff
@@ -54,8 +54,8 @@ export async function main(event, context) {
     return false;
   }
 
-  const { completionFormId } = vivaCaseSSMParams;
-  if (caseItem.currentFormId !== completionFormId) {
+  const { randomCheckFormId, completionFormId } = vivaCaseSSMParams;
```
Comment (karlknowit) [inline services/viva-ms/lambdas/submitCompletion.js:57]: **OPTIONAL**

```js
const notCompletionForm = [randomCheckFormId, completionFormId].includes(caseItem.currentFormId) === false

if (notCompletionForm) { ... }
```

**Review by karlknowit — APPROVED** [review]

File: services/viva-ms/lambdas/submitCompletion.js (hunk 1 above)
Comment (dannil76) [reply services/viva-ms/lambdas/submitCompletion.js:57]: will fix

**Review by karlknowit — APPROVED** [review]

**Pull request merged**